src/style.css

```diff
@@ -0,0 +1,171 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body,
+h2,
+h3 {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: var(--background-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --bottom-color: #00acdc;
+  --text-color: #060b35;
+  --grey-text: #616070;
+  --background-color: #fff;
+}
+
+.foto_image {
+  width: 160px;
+  height: 134px;
+  margin-left: 19px;
+  margin-right: 19px;
+  margin-top: 32px;
+}
+
+.title {
+  width: 166px;
+  height: 36px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--text-color);
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 40px;
+}
+
+.product-code {
+  display: flex;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  width: 98px;
+  height: 14px;
+  color: var(--grey-text);
+  margin-left: 16px;
+  margin-top: 4px;
+  margin-right: 84px;
```

Comment:
Why do you need the margin-right here? Just align this element correctly. Let's imagine that the code will change in the future. How will it work?